pkg/routeagent_driver/handlers/ovn/connection.go

```diff
@@ -67,7 +67,7 @@ func (c *ConnectionHandler) initClients(ctx context.Context, newOVSDBClient NewO
 		return errors.Wrap(err, "error getting OVN NBDB database model")
 	}
 
-	c.nbdb, err = c.createLibovsdbClient(ctx, nbdbModel, newOVSDBClient)
+	c.nbdb, err = c.createLibovsdbClient(c.ipFamily, ctx, nbdbModel, newOVSDBClient)
```

Comment:
No need to pass `c.ipFamily` here as `createLibovsdbClient` can access it internally.